xdsl/dialects/builtin.py

```diff
@@ -202,6 +202,7 @@ def from_params(
 
 
 AnyIntegerAttr: TypeAlias = IntegerAttr[IntegerType | IndexType]
+DefaultIntegerAttrType = i64
```

Comment:
This looks like an unrelated change, is it not?

Reply:
I added that when I was just starting, someone told me that the i64 was the "default" integer attribute type, so I just added that there. I can remove it again, if you don't like it.

Reply:
I would probably move that in the parser!

Reply:
Done!